tests/unit/test-util/test.adb.js

```diff
@@ -1129,6 +1131,44 @@ describe('utils/adb', () => {
       );
     });
 
+    it('informs the user if remote debugging is not enabled', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => {
+            return Promise.resolve(Buffer.from(''));
+          }),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+
+      const maxDiscoveryTime = 50;
+      const retryInterval = 10;
+
+      consoleStream.stopCapturing();
+      consoleStream.flushCapturedLogs();
+      consoleStream.startCapturing();
+      consoleStream.makeVerbose();
+
+      const promise = adbUtils.discoverRDPUnixSocket(
+        'device1', 'org.mozilla.firefox_mybuild', {
+          maxDiscoveryTime,
+          retryInterval,
+        }
+      );
+      const {capturedMessages} = consoleStream;
+
+      assert.ok(capturedMessages.some(
+        (message) => (
+          message.match('[debug]') &&
```

Comment:
Surprisingly this assertion doesn't seem to be fully working, the log we are expecting is emitted with `log.info` and so this test should actually fail and pass if we use '[info]' instead.

Another thing that we may also take care of is to make it clear when this test fails if the message wasn't found or wasn't emitted with the right log level, at the moment the two conditions are checked by a single assertion and so a failure would make it immediately clear.

Let's change this to get the message that we want using something like `const foundMessage = consoleStream.caputedMessages.find((message) => message.includes(expectedMessage))`, then check the two conditions in separate assertions:

```
      assert.ok(foundMessage);
      assert.ok(foundMessage &&
        foundMessage.includes(expectedLogLevel));
```

(NOTE: the `foundMessage && foundMessage.includes` is mainly needed to make the flow type checker happy ;-))